tool/release.sh

```diff
@@ -33,10 +33,11 @@ fi
 
     rm repc.zip
     rm -rf pkg
-    wasm-pack build --profiling -t web -- --no-default-features
+    wasm-pack build --debug -t web -- --no-default-features
```

Comment:
The debug build is 4MB or so. I thought you intentionally wanted to do a profiling build here?

Reply:
I did, but the two builds are the exact same size so it doesn't seem like profiling is doing what we think it is. The debug build compressed is only 50% bigger, so I think we should choose it instead afterall if we're going to have a debug build.

Reply:
profiling only includes the symbol table as far as I can tell... Otherwise it is the same.

I approve of using debug though but it does make profiling harder....

Reply:
> but it does make profiling harder

Oh right, duh.

The thing that's confusing to me is why the profiling and release builds are exactly the same size. IIRC, the release build does lack symbols when I look at in web inspector, while the profiling build has them. But that doesn't match the file sizes.

I'll have to look at this a little more.